XamlAnimatedGif.Shared/AnimationBehavior.cs

```diff
@@ -416,6 +419,9 @@ private static Uri GetAbsoluteUri(Image image)
                 var baseUri = ((IUriContext)image).BaseUri;
 #elif WINRT
                 var baseUri = image.BaseUri;
+#elif SILVERLIGHT 
+                return uri;
+                Uri baseUri = null;
```

Comment:
Not that it really matters, but it seems weird to have a declaration just after a `return`... would you mind moving it before the return? or perhaps just declare the variable before the `#if` blocks.